webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/WCIFHandler.java

```diff
@@ -0,0 +1,195 @@
+package net.gnehzr.tnoodle.server.webscrambles;
+
+import static net.gnehzr.tnoodle.utils.GsonUtils.GSON;
+import static net.gnehzr.tnoodle.utils.GwtSafeUtils.azzert;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+public class WCIFHandler {
```

Comment:
nit: Can we call this something other than `WCIFHandler`? In TNoodle, `*Handler` means "this is a servlet that handles api requests", which this class is not. Maybe call this `WCIFHelper` instead?

Reply:
Renamed